### PR TITLE
add missing package chalk

### PR DESCRIPTION
**Pull request by agnes512**
fixes https://github.com/strongloop/loopback-connector-postgresql/issues/412

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-connector-postgresql) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](https://loopback.io/doc/en/contrib/style-guide-es6.html)
- [ ] Commit messages are following our [guidelines](https://loopback.io/doc/en/contrib/git-commit-messages.html)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
   "dependencies": {
     "async": "^0.9.0",
     "bluebird": "^3.4.6",
+    "chalk": "3.0.0",
```
Comment (raymondfeng) [inline package.json]: Let's use a version range - `"chalk": "^3.0.0",`

**Comment by agnes512** [comment]
@slnode test please

**Comment by agnes512** [comment]
@slnode test please

**Comment by agnes512** [comment]
@slnode test please

**Comment by agnes512** [comment]
@slnode test please

**Comment by agnes512** [comment]
Jenkins fails randomly.

**Pull request merged**